drivers/media/i2c/imx296.c

```diff
@@ -461,6 +465,10 @@ static int imx296_ctrls_init(struct imx296 *sensor)
 				     ARRAY_SIZE(imx296_test_pattern_menu) - 1,
 				     0, 0, imx296_test_pattern_menu);
 
+	v4l2_ctrl_new_int_menu(&sensor->ctrls, NULL, V4L2_CID_LINK_FREQ,
```

Comment:
The ```v4l2_ctrl_handler_init(&sensor->ctrls, 9);``` line hasn't been updated to advise the of the extra control. Did it get updated with the HFLIP and VFLIP patch as well?

Reply:
Will fix.